opsdroid/extern/image_size/README.rst

```diff
@@ -0,0 +1,121 @@
+
```

Comment:
Adding an external dependency as code in the core project is not ideal. I'm assuming the original author has not made this available on PyPI.

My preference would be to create a separate repo for this in the opsdroid org and push it to PyPI as a separate package, then include it in opsdroid with pip.

Reply:
I opened this: https://github.com/scardine/image_size/issues/13 to ask for a version on PyPI, so I am hoping this is just a stop-gap